eras/alonzo/impl/src/Cardano/Ledger/Alonzo/Rules/Utxos.hs

```diff
@@ -221,7 +222,7 @@ scriptsNotValidateTransition = do
   case collectTwoPhaseScriptInputs ei sysSt pp tx utxo of
     Right sLst ->
       case evalScripts @era tx sLst of
-        Passes -> False ?!## ValidationTagMismatch (getField @"isValid" tx) PassedUnexpectedly
+        (Passes _) -> False ?!## ValidationTagMismatch (getField @"isValid" tx) PassedUnexpectedly
         Fails _sss -> pure ()
```

Comment:
This is where we could place an event for those transactions that are successfully submitted to the chain which have a failing script and are marked as `IsValid = False`. `_sss` will collect only the failures, given the `<>` in this PR. we could redo how it all works to collect successes and failures.

Reply:
I do wonder whether this is a symptom of what I mentioned above, which is that it looks like the events and the result of checking are communicated in the same band...